plugins/search/src/components/SearchBar/SearchBar.tsx

```diff
@@ -147,11 +147,20 @@ export type SearchBarProps = Partial<SearchBarBaseProps>;
  * @public
  */
 export const SearchBar = ({ onChange, ...props }: SearchBarProps) => {
```

Comment:
What's your thinking on capturing in `SearchBar` vs. `SearchBarBase`?

Reply:
Hmm mostly because its recommended when using the search context which was my need here. But if we have a need for other things - I can definitely move it to `SearchBarBase`